crates/sui-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -147,6 +153,51 @@ impl<'a> LimitsTraversal<'a> {
         Ok(())
     }
 
+    fn check_multiget_queries(
```

Comment:
This should not be its own traversal, this logic should be integrated into the traversal done by `check_output_limits`, otherwise we will not capture multi-gets that are nested within other fields (e.g. imagine a paginated field that has a multi-get nested within it).

We do not have such cases today, but we have plans to embed the `Query` type within, e.g. `TransactionBlockEffects` (so you can write a query that reads from the state of the network after that transaction has run), and that would introduce such cases.